net/gq/Portfile

```diff
@@ -21,7 +21,13 @@ checksums       rmd160  4e8bf1700f4d7aacd906559e01a11654f6ce0f90 \
 
 openssl.branch  1.0
 
-depends_build   port:intltool \
+use_autoreconf  yes
+
+depends_build-append \
+                port:autoconf \
+                port:automake \
+                port:intltool \
+                port:libtool \
```

Comment:
The autotools deps are added automatically by `use_autoreconf  yes`.

Reply:
Do we still need `intltool` there?

Reply:
Yes.

Reply:
Good now? Check passed.